### PR TITLE
Use Join instead of concat it manually in cephfs.

### PR DESCRIPTION
**Pull request by choury**
Use Join instead of concat it manually in cephfs.

**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Use Join instead of concat it manually in cephfs.
**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/volume/cephfs/cephfs.go [hunk 1]
```diff
@@ -390,22 +382,11 @@ func (cephfsVolume *cephfs) execFuseMount(mountpoint string) error {
 		keyringFile = cephfsVolume.secretFile
 	}
 
-	// build src like mon1:6789,mon2:6789,mon3:6789:/
```
Comment (cwdsuzhou) [inline pkg/volume/cephfs/cephfs.go]: You may need keep the comments.

**Review by cwdsuzhou — COMMENTED** [review]
LGTM with minor change suggestion

File: pkg/volume/cephfs/cephfs.go (hunk 1 above)
Comment (jsafrane) [reply pkg/volume/cephfs/cephfs.go]: yes, please keep the comments, they're useful.

**Comment by choury** [comment]
@jsafrane  Would you please add a lgtm label to merge this pr? thanks.

**Pull request merged**